Edits on the Enable CORS topic (#4942)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/enable-cors/main/index.md

* Grant cross-origin access to the Okta API from your web apps.
* Test your Okta CORS configuration.

> #### What you need

Reviewer: ```suggestion
#### What you need

```

Reviewer: Should have a space after the heading.

An [Okta Developer Edition org](https://developer.okta.com/signup/)

---

## About CORS

[Cross-Origin Resource Sharing (CORS)](https://www.w3.org/TR/cors/) is a mechanism that allows a web page to make an AJAX call using [XMLHttpRequest (XHR)](https://xhr.spec.whatwg.org/).

> You can use XHR to call a domain that is different from the domain where the script was loaded. Web browsers typically don't allow cross-domain requests because of the [same origin security policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy). However, CORS defines a standardized way for the server and browser to interact. Then CORS determines whether to allow a cross-origin request.

Reviewer 2: ```suggestion
You can use XHR to call a domain that's different from the domain where the script was loaded. Web browsers typically don't allow cross-domain requests because of the [same origin security policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy). However, CORS defines a standardized way for the server and browser to interact. Then CORS determines whether to allow a cross-origin request.
```

> ### CORS and Okta

Reviewer: I like this edit. A good way to break down the info.

* In Okta, CORS allows JavaScript, which is hosted on your website, to make an XHR to the Okta API with a token. See [OAuth 2.0](/docs/guides/implement-oauth-for-okta/).
* Every website origin must be explicitly permitted through the Admin Console for CORS. See [Grant cross-origin access to websites](#grant-cross-origin-access-to-websites).
* If you’re using [OAuth 2.0](/docs/guides/implement-oauth-for-okta/) tokens to make calls to Okta APIs, you don't need to add a Trusted Origin. OAuth for Okta APIs doesn't rely on cookies but uses bearer tokens instead. See [Scopes and supported endpoints](/docs/guides/implement-oauth-for-okta/main/#scopes-and-supported-endpoints).

> **Caution:** Only grant access to specific origins (websites) that you control and trust to access the Okta API.

### API Support

> The Okta API supports CORS on an API by API basis. If you're building an application that needs CORS, check that the specific operation supports CORS for your use case. APIs that support CORS are marked with the following icon <span class="api-label api-label-small api-label-cors"><i class="fa fa-cloud-download"></i>CORS</span>.

Reviewer 2: ```suggestion
The Okta API supports CORS on an API by API basis. If you're building an app that needs CORS, check that the specific operation supports CORS for your use case. APIs that support CORS are marked with the following icon <span class="api-label api-label-small api-label-cors"><i class="fa fa-cloud-download"></i>CORS</span>.
```
